### ISSUE TITLE
nimble dump wants an answer

### ISSUE THREAD

**Issue by FedericoCeratto** [nim-lang/nimble#640]
Nimble dump gets stuck asking for an answer when run against https://github.com/FedericoCeratto/nim-syslog/ at tag 0.0.3

```
$ nimble dump 
    Answer: 
```

nimble dump -n  prints nothing
nimble dump -y  prints the expected dump

**Comment by jrfondren** [comment]
For reference this is the .nimble file that caused the problem:
```
[Package]
name          = "syslog"
version       = "0.0.3"
author        = "Federico Ceratto"
description   = """Minimal syslog support."""
license       = "LGPLv3"

installDirs = """


"""

InstallFiles = """

LICENSE
README.md
syslog.nim

"""

[Deps]
Requires: """

nake >= 1.4
nim >= 0.10.2

"""
```

**Comment by genotrance** [comment]
Is this fixed in #648?
